Gordon360/ApiControllers/HousingController.cs

```diff
@@ -103,5 +103,18 @@ public IHttpActionResult ChangeEditor([FromBody] ApartmentAppNewEditorViewModel
 
             return Ok(result);
         }
+
+        /// <summary>
+        /// Export the SQL data into CSV file
+        /// </summary>
+        /// <returns>CSV file with an exported data</returns>
+        [HttpGet]
+        [Route("csv")]
```

Comment:
Should probably change to "apartment/csv" to match the new naming scheme
```suggestion
        [Route("apartment/csv")]
```

Reply:
Good catch. Commited the change.